docs/release-notes/8.9.asciidoc

```diff
@@ -147,6 +147,7 @@ There are no breaking changes in 8.9.0.
 [discrete]
 [[features-8.9.0]]
 ==== New features
+* Enables the `top` command for MacOS and Linux, which shows active processes sorted by much CPU they are causing {elastic-endpoint} to consume (https://www.github.com/elastic/endpoint/blob/main/EndpointTopCommand.md[Readme]).
```

Comment:
Just a few suggestions and one question:
- Switched the verb to "introduces" because "enables" implies that something can be turned on or off. Feel free to reject this change if you can actually enable/disable this command.
- WDYT about adding a bit more info about the results that the `top` command returns? See my edits below. 
- Do we need to mention the `limit` [param](https://github.com/elastic/security-docs/issues/3889#issuecomment-1741434722) or will this be eventually added to the [reamde](https://www.github.com/elastic/endpoint/blob/main/EndpointTopCommand.md)? 

```suggestion
* Introduces the `top` command for MacOS and Linux, which shows active processes that {elastic-endpoint} is monitoring and recording. Processes are sorted by how much CPU they are causing {elastic-endpoint} to consume. Learn more about the `top` command by referring to the (https://www.github.com/elastic/endpoint/blob/main/EndpointTopCommand.md[Readme]).
```

Reply:
Thanks, great ideas. I think the limit command will make more sense in the readme. Plus it's for 8.11 so if anything we would release note it in the next release notes!

Reply:
Agree! I left a [note](https://github.com/elastic/security-docs/issues/4015) about release-noting the `limit` param in the 8.11 release notes so we/I don't forget : )